src/components/SelectedStudentsActionWrapper.js

```diff
@@ -19,10 +105,23 @@ class SelectedStudentsActionWrapper extends Component{
           </CSVLink>
         </div>
         <div className="buttonContainer">
-          <button className="linkButton">
+          <button className="linkButton" onClick={this.openPrintOption}>
               <i className="fa fa-print card-icon"></i>Print Now
           </button>
         </div>
+        <Modal
+          isOpen={this.state.printOptionIsOpen}
+          onAfterOpen={this.afterOpenModal}
+          onRequestClose={this.closePrintOption}
+          style={customPrintOptionStyles}
+        >
+          <div > {this.renderStudentIdCards(this.props.selectedStudents)} </div>
```

Comment:
```suggestion
          <div>{this.renderStudentIdCards(this.props.selectedStudents)}</div>
```

Make sure avoid any extra white spaces in the code